config.json

```diff
@@ -28,6 +28,20 @@
         "strings"
       ]
     },
+    {
+      "slug": "high-scores",
+      "uuid": "574d6323-5ff5-4019-9ebe-0067daafba13",
+      "core": true,
+      "unlocked_by": null,
```

Comment:
Should be unlocked by something. I think `two-fer` is a good one to unlock this.